.github/workflows/end2end.yml

```
@@ -0,0 +1,60 @@
+name: End-to-end test of EasyBuild in different distros
+on: [push, pull_request]
+jobs:
+  build_publish:
+    name: End-to-end test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        container:
+          - centos-7.9
+          - centos-8.5
+          - fedora-36
+          - opensuse-15.4
+          - rockylinux-8.7
```

Maybe we can also test on one Rocky 9 version?
```suggestion
          - rockylinux-8.7
          - rockylinux-9.1
```
I also opened an MR to add Rocky Linux 8.8 and 9.2 in case we want to update: https://github.com/easybuilders/easybuild-containers/pull/24

---

Yeah let's make it 8.8. and 9.2

```suggestion
          - rockylinux-8.8
          - rockylinux-9.2